package.json

```diff
@@ -35,7 +35,6 @@
     "pretest": "node tasks/pretest.js",
     "test-jasmine": "karma start test/jasmine/karma.conf.js",
     "test-image": "node tasks/test_image.js",
-    "test-image-gl2d": "node tasks/test_image.js gl2d_* --queue",
```

Comment:
Nice find! 

Can you :zap: that `npm run test-image-gl2d` piece in:

```
    "test": "npm run test-jasmine && npm run test-bundle && npm run test-image && npm run test-image-gl2d && npm run test-syntax && npm run lint",
```

Reply:
and actually `test` should be:

```js
"test": "npm run test-jasmine -- --nowatch && npm run test-bundle && npm run test-image && npm run test-export && npm run test-syntax && npm run lint",
```

I think anybody actually uses this command, but let's keep it up-to-date for documentation purposes.

Reply:
Done in be336489a93f6600c6ac668f4f7b46b323d99e53